tests/tracking/test_tracking.py

```diff
@@ -451,9 +451,9 @@ def test_set_tags():
 def test_log_metric_validation():
     with start_run() as active_run:
         run_id = active_run.info.run_id
-        with pytest.raises(MlflowException, match="Got invalid value apple for metric") as e:
+        with pytest.raises(MlflowException):
```

Comment:
Ideally, it should be up to the backend to determine the validity of a metric value. If our attempted conversion fails, instead of throwing, can we propagate the original metric value as part of the request?